tests/test_pipelines.py

```diff
@@ -258,40 +266,17 @@ def test_tf_feature_extraction(self):
             self._test_mono_column_pipeline(nlp, valid_inputs, invalid_inputs, {})
 
     @require_torch
-    def test_fill_mask(self):
+    def test_torch_fill_mask(self):
         mandatory_keys = {"sequence", "score", "token"}
         valid_inputs = [
             "My name is <mask>",
             "The largest city in France is <mask>",
         ]
         invalid_inputs = [None]
```

Comment:
removed expected_multi_result for speed.

Reply:
can we keep those integration tests in `@slow`?